packages/nimble-components/src/anchor/specs/README.md

```diff
@@ -0,0 +1,203 @@
+# Nimble Anchor
+
+## Overview
+
+The Nimble Anchor is a component used to navigate to a web resource, similar to the HTML anchor element.
+
+### Background
+
+[Visual Design spec](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46)<br>
+[Nimble work item](https://github.com/ni/nimble/issues/324)
+
+---
+
+## Design
+
+**Anchors in other components**
+
+Initially we will create two flavors of anchor: a standard anchor, and a button-like anchor. In the future, we expect to support anchors within other controls, e.g.
+
+-   tabs
+-   menu
+-   tree
+
+Unfortunately it's not as simple as dropping `nimble-anchor`s into the existing components. Each of these has its own considerations and challenges.
+
+-   **Tabs**
```

Comment:
The detail about the challenges with tabs, menus, and trees is awesome but since it's still less fully baked than the rest of the spec (contains a lot of open questions/hypotheses/options) I find it makes it harder to read and approve the rest of the spec. Could you consider one of:
1. pulling it into a separate PR
2. pulling it into a separate document within this PR
3. moving it to a section at the bottom of this document with a title like **Future directions** and a disclaimer that it's just notes for future reference, not decisions